resources/setup_emulator.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+kill_all_emus() {
+    for emu_device in $(adb devices -l |grep 'device product:' |cut -d' ' -f1); do
+        echo "Killing emulator: $emu_device"
+        adb -s $emu_device emu kill
+    done
+}
+
+boot_finished_status="stopped"
+attempt=0
+timeout=300
+
+echo 'Creating emulator'
+echo 'n' | $ANDROID_HOME/tools/bin/avdmanager create avd -n test-android -f -k "system-images;android-27;google_apis;x86" 
+
+echo 'loading emulator'
+$ANDROID_HOME/emulator/emulator -avd test-android -no-boot-anim -no-window -noaudio & 
+
+while true; do
```

Comment:
For readability, can you indent the loop?

Reply:
done